### PR TITLE
Document improvements in the Change Log

### PR DESCRIPTION
**Pull request by JackStouffer**
The change log is as much a marketing document as it is a historical one. These changes should be promoted in order to get people to download this new version.

### CODE REVIEW

**Comment by quickfur** [comment]
LGTM, thanks!

**Comment by JakobOvrum** [comment]
So, these are changes that were published in 2.070, and you're suggesting we add them to the changelog for 2.071?

We can still fix the changelog for the older versions, there's no need to sow confusion by pretending that changes are newer than they actually are. Marketing isn't the only use for a changelog. Someone might look at the changelog to evaluate which version of the compiler they need to support, or even which compiler they can use for their project. Retroactively lumping old features into the changelog for the most recent version is a disservice to the porting efforts done by the GDC and LDC teams.

**Comment by JackStouffer** [comment]
> So, these are changes that were published in 2.070, and you're suggesting we add them to the changelog for 2.071?

I'm sorry if they were already released, I was going off this list: https://github.com/D-Programming-Language/phobos/compare/v2.070.0...master

I thought that showed all of the changes between 2.070 and now.

**Comment by JakobOvrum** [comment]
No, I'm simply wrong. Sorry.

LGTM and thanks!

**Comment by quickfur** [comment]
Auto-merge toggled on

**Comment by JackStouffer** [comment]
Thanks

**Comment by MartinNowak** [comment]
Thanks, would've been great if you'd actually tested this.
There is a changelog.html target in the makefiles of dmd, druntime, and phobos.

File: changelog.dd [hunk 1]
```diff
@@ -15,7 +15,13 @@ $(LI $(XREF std_meta, Repeat) was added to obtain a repeating
...
 $(LI $(D fold) was added as an alternative to $(D reduce) with argument order swapped.)
 $(LI $(RELATIVE_LINK2 nextPow2, Added nextPow2 and truncPow2 to std.math.))
-
+$(LI $(REF bsf core,bitop), $(REF bsr core,bitop), and $(REF popcnt, core,bitop) now
+work with ulongs on 32 bit systems.)
+$(LI $(REF topN std,algorithm,sorting) is now 5% - 7% faster.)
+$(LI $(RELATIVE_LINK2 unary, Unary overloads of $(REF startsWith std,algorithm,searching) and
+$(REF endsWith std,algorithm,searching) were added.))
+$(LI $(RELATIVE_LINK2 maxCount, $(REF maxCount std,algorithm,searching) and
+$(REF maxPos std,algorithm,searching) were added.))
```
Comment (MartinNowak) [inline changelog.dd]: Should've been `CXREF` and `XREF`, see the macros below.

Comment (JakobOvrum) [inline changelog.dd]: How is the changelog generated? `REF` is in dlang.org.ddoc and invoked correctly here.

**Pull request merged**